tests/containers/validate_btrfs.pm

```diff
@@ -65,15 +66,14 @@ sub _test_btrfs_device_mgmt {
     script_run("df -h");
     # Determine the remaining size of /var
     my $var_free = script_output('df 2>/dev/null | grep /var | awk \'{print $4;}\'');
-    # /var is using its own partition. Create file in the container enough to fill up the partition up to 99%
-    my $fill = int($var_free * 0.99 * 1024);    # df returns the size in KiB
+    # Create file in the container enough to fill the "/var" partition (where the container is located) up to 90%
+    # Note: Don't fill it up more than 90% otherwise `btrfs balance` might stumble.
+    my $fill = int($var_free * 1024 * 0.90);    # df returns the size in KiB
     $rt->up('huge_image', keep_container => 1, cmd => "fallocate -l $fill bigfile.txt");
-    validate_script_output "df -h --sync|grep var", sub { m/\/dev\/vda.+\s+(9[7-9]|100)%/ };
+    validate_script_output "df -h --sync|grep var", sub { m/\/dev\/vda.+\s+(9[0-9]|100)%/ };
```

Comment:
Shouldn't we use rather `btrfs fi df`?

Reply:
`btrfs fi df` is used in the next lines, see line 75-76. IMHO it is safe to use `df -h` here, because we are allocating a single BIG file, where there should not be any meaningful difference between `btrfs fi df` and `df`. So I'd say it is safe to keep it this way.

Reply:
No hard feeling, I'm resolving this. I just heard it is recommended to not use `df` with `btrfs` as it has 'additional features' :)